docs/using-gitbase/functions.md

```diff
@@ -19,6 +19,17 @@ To make some common tasks easier for the user, there are some functions to inter
 |`uast_children(blob) blob`| returns a flattened array of the children UAST nodes from each one of the UAST nodes in the given array.              |
 |`loc(path, blob) json`| returns a JSON map, containing the lines of code of a file, separated in three categories: Code, Blank and Comment lines. |
 |`version() text`| returns the gitbase version in the following format `8.0.11-{GITBASE_VERSION}` for compatibility with MySQL versioning. |
+||||||| merged common ancestors
+|`is_remote(reference_name)bool`| check if the given reference name is from a remote one                                                          |
+|`is_tag(reference_name)bool`| check if the given reference name is a tag                                                                         |
+|`language(path, [blob])text`| gets the language of a file given its path and the optional content of the file                                    |
+|`uast(blob, [lang, [xpath]]) blob`| returns a node array of UAST nodes in semantic mode                                                          |
+|`uast_mode(mode, blob, lang) blob`| returns a node array of UAST nodes specifying its language and mode (semantic, annotated or native)          |
+|`uast_xpath(blob, xpath) blob`| performs an XPath query over the given UAST nodes                                                                |
+|`uast_extract(blob, key) text array`| extracts information identified by the given key from the uast nodes                                       |
+|`uast_children(blob) blob`| returns a flattened array of the children UAST nodes from each one of the UAST nodes in the given array              |
+|`loc(path, blob) json`| returns a JSON map, containing the lines of code of a file, separated in three categories: Code, Blank and Comment lines |
```

Comment:
We should add a period at the end of the descriptions to be consistent with the rest of rows.

Reply:
Added periods everywhere.